protos/plugin/codegen.proto

```diff
@@ -68,6 +68,7 @@ message PythonCode
   bool emit_async_querier = 3;
   string package = 4;
   string out = 5;
+  bool use_pydantic_models = 6;
```

Comment:
To match the other configuration options, rename this to `emit_pydantic_models`.